python_modules/dagster/dagster/_cli/api.py

```diff
@@ -315,7 +315,7 @@ def verify_step(instance, pipeline_run, retry_state, step_keys_to_execute):
         "interactively."
     ),
 )
-@click.argument("input_json", type=click.STRING)
+@click.argument("input_json", type=click.STRING, envvar="DAGSTER_EXECUTE_STEP_ARGS")
```

Comment:
im a little nervous about using a new manually set env var rather than the one implicitly provided by click - i guess we should never be out of version sync on the two sides of this exchange but still spooks me

Reply:
Click only implicilty provides env for options, not args https://click.palletsprojects.com/en/7.x/arguments/#environment-variables

In terms of backcompat... yeah originally I made this opt in but it seemed rare enough that run and step workers are different images (certainly doesn't happen by default)